src/main/java/org/jenkinsci/plugins/saml/SamlFormValidation.java

```diff
@@ -58,9 +58,6 @@ public static FormValidation checkEmailFormat(String value, String message) {
         catch (AddressException ae) {
             return FormValidation.error(message);
         }
-        catch (NullPointerException ne) {
-            return FormValidation.warning(message);
-        }
```

Comment:
This whole method appears to be dead code, but it trips up new versions of SpotBugs. Anyway it does not appear that a `NullPointerException` could actually occur here even if this method was invoked.